src/app/record/components/org-identifier/org-identifier.component.html

```diff
@@ -66,9 +66,34 @@ <h3 class="orc-font-body-small">
       class="padding-level-3"
     >
       {{ org.identifierType }}:
-      <ng-container *ngIf="org.preferred && !isUrl(org.preferred)">{{
-        org.preferred
-      }}</ng-container>
+      <ng-container *ngIf="org.preferred && !isUrl(org.preferred)">
+        <ng-container
+          *ngIf="
+            getLink(org.identifierType, org.preferred) !== '';
+            else plainUrl
+          "
+        >
+          <a
+            class="underline"
+            target="_blank"
+            rel="noopener noreferrer"
+            href="{{ getLink(org.identifierType, org.preferred) }}"
+          >
+            {{ org.preferred }}
+          </a>
+        </ng-container>
+        <ng-template #plainUrl>
+          <a
+            *ngIf="isUrl(org.preferred)"
```

Comment:
seems like `*ngIf="isUrl(org.preferred)"` will never be true, since is wrap with another  if `!isUrl(org.preferred)`